lib/Common/Core/Assertions.h

```diff
@@ -15,9 +15,11 @@ _declspec(selectany) int AssertsToConsole = false;
 
 #if _WIN32
 _declspec(thread, selectany) int IsInAssert = false;
+#elif !defined(__IOS__)
+__declspec(thread, selectany) int IsInAssert = false;
```

Comment:
Oh, I guess thread-local is not yet supported on iOS? Ok. Curious, what is the difference between _declspec and __declspec? If they're the same, can we just combine the _WIN32 and !defined(__IOS__) cases?